nvtabular/tag.py

```diff
@@ -0,0 +1,58 @@
+from enum import Enum
+
+
+class DefaultTags(Enum):
+    # Feature types
+    CATEGORICAL = ["categorical"]
+    CONTINUOUS = ["continuous"]
+    LIST = ["list"]
+    IMAGE = ["image"]
+    TEXT = ["text"]
+    TEXT_TOKENIZED = ["text_tokenized"]
+    TIME = ["time"]
```

Comment:
For the moment, I think we only support `CATEGORICAL`, `CONTINUOUS`, and `LIST`, so we might want to leave these other tags for later?

Reply:
We have some code in Transformers4Rec that uses `TEXT` & `TEXT_TOKENIZED`. I don't see an issue of having some often used tags here that don't have direct usage in our library code from the start.